lib/rouge/lexers/c.rb

```diff
@@ -166,7 +166,7 @@ def self.builtins
         rule %r/\\([\\abfnrtv"']|x[a-fA-F0-9]{2,4}|[0-7]{1,3})/, Str::Escape
         rule %r/[^\\"\n]+/, Str
         rule %r/\\\n/, Str
-        rule %r/\\/, Str # stray backslash
+        rule %r(\\), Str # stray backslash
```

Comment:
Is this change intentional? There are a couple more like it throughout the docs.

Reply:
What's wrong in this change?

Reply:
oh.. I see it.. The change is unnecessary...